core/app/models/spree/app_configuration.rb

```diff
@@ -117,10 +117,15 @@ class AppConfiguration < Preferences::Configuration
     preference :currency, :string, default: "USD"
 
     # @!attribute [rw] default_country_id
-    #   @deprecated
+    #   @deprecated Use the default country ISO preference instead
     #   @return [Integer,nil] id of {Country} to be selected by default in dropdowns (default: nil)
     preference :default_country_id, :integer
 
+    # @!attribute [rw] default_country_iso
+    #   @deprecated
+    #   @return [Integer,nil] id of {Country} to be selected by default in dropdowns (default: nil)
```

Comment:
Yep, you're right. The comment also says deprecated, which I don't intend
it to be. I'll force push once the build is through.
Am 07.02.2016 19:28 schrieb "Thomas von Deyen" <notifications@github.com>:

> In core/app/models/spree/app_configuration.rb
> <https://github.com/solidusio/solidus/pull/810#discussion_r52121680>:
>
> >      #   @return [Integer,nil] id of {Country} to be selected by default in dropdowns (default: nil)
> >      preference :default_country_id, :integer
> >
> > +    # @!attribute [rw] default_country_iso
> > +    #   @deprecated
> > +    #   @return [Integer,nil] id of {Country} to be selected by default in dropdowns (default: nil)
>
> The comment says default nil, but the code 'us'
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/solidusio/solidus/pull/810/files#r52121680>.
>